docs/cfp/cntug-cfp-en.md

```diff
@@ -0,0 +1,96 @@
+---
+id: cfp-en
+title: Call for Proposals
+tags:
+  - cfp
+  - cntug-cfp
+  - en
+description: CFP for Cloud Native Taiwan User Group Meetup
+---
+
+# Call for Proposals (CFP)
+
+Last Updated: 20230817
+Version: English
+
+## Submit Your Talk {#submit-cfp}
+
+Please fill in [Cloud Native Taiwan User Group Meetup Call for Speaker][7], or
+
+```markdown title="submit-cfp"
+- Speaker Name:
+- Speaker Email:
+- Speaker photos (Optional):
+- Speaker phone (Optional):
+- Speaker Organization / Job title:
+- Speaker Personal Profile:
+- Agenda topic:
+- Agenda introduction:
+- Language of the agenda: `Mandarin Chinese` / `English`
+- The duration of your agenda (mins): `60` / 15
+- Location: `Online` / In-person / Both
+- Video recorded on YouTube: `YES` / no
+```
+
+## Suggested Topics
+
+Needs to be related to [CNCF Landscape][1] and Open Infra Foundation related project
```

Comment:
Not sure if it will be good to include broader range of projects like Ceph.
I think it might be good to say Software Infrastructure related projects.